test/tests/functional/pbs_test_tpp.py

```diff
@@ -825,6 +829,149 @@ def test_COMM_ROUTERS_with_nondefault_pbs_conf(self):
                           job=True, interactive=True, resv=True,
                           resv_job=True)
 
+    def start_gdb(self, comm_pid):
+        """
+        This function starts gdb and attach it to pbs_comm
+        :param comm_pid: Comm's Pid
+        :type comm_pid: Integer.
+        """
+        gdb_python = """
+import gdb
+gdb.execute("attach %s")
+o = gdb.execute("info threads", to_string=True)
+print(o)
+gdb.execute("detach")
+gdb.execute("quit")
+quit()
+""" % comm_pid
+        fn = self.du.create_temp_file(body=gdb_python)
+        # Catch the stack trace using gdb
+        gdb_cmd = ["gdb", "-P", fn]
+        out_path = os.path.join(self.tmp_dir, "gdb.out")
+        with open(out_path, "w") as outfd:
+            self.du.run_cmd(cmd=gdb_cmd, stdout=outfd, stderr=STDOUT,
+                            sudo=True)
+        gdb_out = ""
+        f = open(out_path, "r")
+        for lines in f:
+            gdb_out += lines
+        num_wait_thread = gdb_out.count("pwait")
+        return num_wait_thread
+
+    def test_PBS_COMM_THREADS(self):
+        """
+        Test allowable values for variable PBS_COMM_THREADS of pbs.conf
```

Comment:
PBS_COMM_THREADS is not a variable but a conf parameter. Let us just say
        Test allowable values for PBS_COMM_THREADS